ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivityBuilder.java

```diff
@@ -77,7 +77,7 @@ protected void extendUIActivity(BaseUIActivity uiActivity, ExoSocialActivity act
     // So we couldn't set activity.getTitle() all the time, see INTEG-486
     if (activity.getTemplateParams() != null
         && StringUtils.isNotBlank(templateParams.get(FileUIActivity.MESSAGE))) {
-      fileActivity.setMessage(templateParams.get(FileUIActivity.MESSAGE));
+      fileActivity.setMessage(templateParams.get(FileUIActivity.MESSAGE).split("<oembed>")[0]);
```

Comment:
You need check if the message contains that tag before getting the first element otherwise you will get an ArrayOutOfBoundsException